### ISSUE TITLE
CosignServiceImplTest.java listBuilderCosignForUser() method needs cleanup

### ISSUE THREAD

**Issue by haxwell** [savvato-software/tribe-app-backend#193]
CosignServiceImplTest.java, the method listBuilderCosignForUser().. 

1. `Long userIdReceiving = userPerPhraseCount + 1L;` -- Why is the userIdReceiving being based on a count? Just set it to something, don't base it on this variable. Someone coming behind (like me) looking at this won't understand why it's being done.

2. 
```
for(int j=1; j<=userPerPhraseCount; j++)
  cosign.setUserIdIssuing((long)j);
cosign.setUserIdReceiving(userIdReceiving);
```
- Why are we repeatedly setting UserIdIssuing, but userIdReceiving we set just once? Both should be set just once.

**Comment by mrsbluerose** [comment]
@haxwell The helper method I built is a little unclear and admittedly outdated since we are trying to use testing constants now. Here's what I did:
- you requested I test more than one result for the getAllCosignsForUser(), because we wanted to make sure it was performing appropriately for multiple results (I originally only tested one result): [https://github.com/savvato-software/tribe-app-backend/pull/162#discussion_r1517700490](https://github.com/savvato-software/tribe-app-backend/pull/162#discussion_r1517700490)
- It is a long test, comparatively speaking, and I condensed the part that required creating 9 cosigns into this helper method that creates a list of 9 cosigns.
- point 1 above: `Long userIdReceiving = userPerPhraseCount + 1L;` ensures that the user ID receiving the cosigns from three other users is never equal to any of the userIssuing IDs. This is based on the user issuing id's being 1, 2, 3 or some sequential numbers of IDs. Now that we are using constants, this should be updated.
- point 2 above: The result was intended to be three different users issuing cosigns for one single user for three different phrases, thus resulting in three cosigns for three phrases for one user. This code was intended to make the test easier to read, but I can see how it may actually make it more complex to dig through.

I could take this ticket and either get rid of the helper method and write out the longer code, or I could rework the helper method with constants and make comments to better explain the logic. What are your thoughts?
